Source/DafnyCore/Verifier/ProofObligationDescription.cs

```diff
@@ -279,7 +279,7 @@ public class PreconditionSatisfied : ProofObligationDescription {
       : $"error is impossible: {customErrMsg}";
 
   public override string FailureDescription =>
-    customErrMsg ?? "function precondition might not hold";
+    customErrMsg ?? "Could not prove function precondition";
```

Comment:
```suggestion
    customErrMsg ?? "could not prove function precondition";
```

The convention in the rest of the messages is for them to start with a lower-case letter.